gulpfile.babel.js

```diff
@@ -1,5 +1,8 @@
 import fs from 'fs';
 import path from 'path';
+import config from './lib/config';
+
+require('dotenv').config({path: config.get('dotenv.file')});
```

Comment:
Not a huge dealbreaker, but since we're using `import`'s everywhere I'd say it would be more consistent to do so here. Or am I missing something? Also it's kind of implicit that `dotenv` will fall back to the the `cwd`, so might make it more specific for future generations? Like so:

```js
import dotenv from 'dotenv';

/**
 * Load environment variables from`.env`
 */
dotenv.config({ path: config.get('dotenv.file') || process.cwd() });
```